htdocs/includes/odtphp/odf.php

```diff
@@ -584,7 +584,7 @@ public function exportAsAttachedPDF($name="")
 			// using windows libreoffice that must be in path
 			// using linux/mac libreoffice that must be in path
 			// Note PHP Config "fastcgi.impersonate=0" must set to 0 - Default is 1
-			$command ='soffice --headless --convert-to pdf --outdir '. escapeshellarg(dirname($name)). " ".escapeshellarg($name);
+			$command ='soffice --headless -env:UserInstallation=file:///tmp/nobody --convert-to pdf --outdir '. escapeshellarg(dirname($name)). " ".escapeshellarg($name);
```

Comment:
By definition, Dolibarr write only inside the documents directory (even temporary files are never stores outside this directory). 
Can you test to replace -env:UserInstallation=file:///tmp/nobody
by
-env:UserInstallation=file:'.$conf->user->dir_temp.'
So the temp directory will be dolibarr temp directory of the current user.

Reply:
Hello edly,

i've double checked it in an Test environment (blank install with standard odt templates). It works with -env:UserInstallation=file:"'.$conf->user->dir_temp.'". How do i change the commit? Sorry this is my first commit to a big repo.

Reply:
Knowing it works is enough for me, so i validate PR and add another commit to make the change.
Next time, to do it yourself, i think all you have to do is to make the change on your dev environment, commit and send another PR.

Reply:
Okay thank you very mouch. Next time i will do it the that way. :-)